src/main/java/com/jagrosh/jmusicbot/settings/RepeatMode.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.settings;
+
+/**
+ *
+ * @author Michaili K
+ */
+public enum RepeatMode
+{
+    Off,
+    On,
+    One
```

Comment:
Shall I make the fields uppercase or keep it like that?

Reply:
They should be uppercase. Let's also add the emojis here, something along the lines of
```java
public enum RepeatMode
{
    OFF(""), ON("\uD83D\uDD01"), ONE("\uD83D\uDD02"); // 🔁, 🔂

    private final String emoji; 

    private RepeatMode(String emoji)
    {
        this.emoji= emoji;
    }

    public String getEmoji()
    {
        return emoji;
    }
}
```